osu.Game/Beatmaps/IWorkingBeatmap.cs

```diff
@@ -59,5 +54,10 @@ public interface IWorkingBeatmap
         /// <returns>The converted <see cref="IBeatmap"/>.</returns>
         /// <exception cref="BeatmapInvalidForRulesetException">If <see cref="Beatmap"/> could not be converted to <paramref name="ruleset"/>.</exception>
         IBeatmap GetPlayableBeatmap(RulesetInfo ruleset, IReadOnlyList<Mod> mods = null, TimeSpan? timeout = null);
+
+        /// <summary>
+        /// Retrieves the <see cref="Track"/> which this <see cref="WorkingBeatmap"/> provides.
```

Comment:
Was about to suggest this reads "Retrieves a new instance of the..." but I think due to trackStore doing caching internally this generally isn't always true. I guess it reads fine as-is.